eth/tools/builder/chain/builders.py

```diff
@@ -290,11 +290,18 @@ def validate_seal(cls, header: BlockHeader) -> None:
 @to_tuple
 def _mix_in_disable_seal_validation(vm_configuration: VMConfiguration) -> Iterable[VMFork]:
     for fork_block, vm_class in vm_configuration:
-        vm_class_without_seal_validation = type(
-            vm_class.__name__,
-            (NoVMSealValidationMixin, vm_class),
-            {},
-        )
+        if issubclass(vm_class, NoVMSealValidationMixin):
+            vm_class_without_seal_validation = type(
```

Comment:
How about we just return `vm_class` unmodified since this is effectively just adding another layer to the inheritance chain which doesn't actually add or modify any functionality.